php/class-wp-cli.php

```diff
@@ -1004,6 +1004,10 @@ public static function error_to_string( $errors ) {
 				return $message;
 			}
 		}
+
+		if ( $errors instanceof \Exception ) {
+			return $errors->getMessage();
```

Comment:
It might make sense to also print the concrete class of the exception, as that is often the most important distinction amongst exceptions.

Reply:
I prefixed the error message with a class name.